omero/developers/Python.rst

```diff
@@ -222,6 +222,37 @@ Read data
             for image in dataset.listChildren():
                 print_obj(image, 4)
 
+-  **Get Objects by their ID or attributes**
+
+   The first argument for ``conn.getObjects()`` or ``conn.getObject()`` is the object type.
+   This is not case sensitive. Supported types are
+   ``project``, ``dataset``, ``image``, ``screen``, ``plate``, ``plateacquisition``, ``acquisition``, ``well``,
+   ``roi``, ``shape``, ``experimenter``, ``experimentergroup``, ``originalfile``, ``fileset``, ``annotation``
+
+::
+
+    # Find objects by ID
+    projects = conn.getObjects("Project", [1, 2, 3])
+
+    # Single object. IDs are unique across all types of annotations
+    image = conn.getObject("Annotation", 1)
+
+    # Find an Object by attribute. E.g. 'name'
```

Comment:
where is the list of other attributes ?

Reply:
You'd need to look at https://docs.openmicroscopy.org/latest/omero/developers/GettingStarted.html#omero-model for a link to e.g. https://docs.openmicroscopy.org/omero-blitz/5.5.3/slice2html/omero/model.html which has attributes for all objects. I'll add a link in...